portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
```

Comment:
Was this intended to be part of this PR? If so, please update the name/description to include this.